src/TokenManagers/XboxTokenManager.js

```diff
@@ -138,6 +139,31 @@ class XboxTokenManager {
     }
   }
 
+  async doSisuAuth(accessToken, deviceToken) {
+		const payload = {
+			AccessToken: 't=' + accessToken,
+			AppId: XboxAppiOS,
```

Comment:
This should specified by the user in the constructor options. Just like titleAuth currently is. If it’s specified, this authentication endpoint can be used instead of the xsts authorize one. 

Also needs lint, “npm run fix”

Reply:
These are the tokens that minecraft.net uses, 
```ts
export const sharedEnvConfig = {
  currency: "",
  language,
  locale,
  "cookie.name": "bearer_token",
  "cookie.options": {
    domain: ".minecraft.net",
    secure: true,
  },
  "enable.dungeonsPreorder": false,
  "enable.msaMigration": true,
  z: function(metadata: import("@mojang/telemetry").IMetadata, data: any) {
    console.log("🔥", metadata, data);
  },
  "url.forgotPassword": location.origin + "/" + language + "/password/forgot/",
  "url.homePage": location.origin + "/",
  "url.login": location.origin + "/" + language + "/login",
  "url.profile": location.origin + "/" + language + "/profile",
  "cookie.entitlementState": "hasMcJavaEntitlement",
  "enable.arkoseCaptcha": false,
  "endpoint.arkose": "https://minecraft-api.arkoselabs.com/",
  msaStateLoginParam: "login",
  "url.supportCenter": "https://help.mojang.com/",
  "url.supportCenterforMSAMigration": "https://help.minecraft.net/hc/en-us/sections/360011399911-Migration-for-Minecraft-Java-Edition",
  "minecraft.entitlements": ["product_minecraft", "game_minecraft"],
  "url.xboxLogoutReturnUrl": location.origin + "/" + language,
  "url.imageBaseUrl": __webpack_public_path__ + "assets",
  titleId: "896928775", /// <<
  sisuAppId: "1142970254", /// <<
} as const;
```
```ts
export const msaUtilsConfig = {
  relyingPartyKey: "rp://api.minecraftservices.com/",
  xboxSisuTestParam: "",
  "url.xboxLoginReturnUrl": location.origin + "/" + language + "/login",
  "url.xboxRegisterReturnUrl": location.origin + "/" + language + "/login",
  sisuCobrandId: "8058f65d-ce06-4c30-9559-473c9275a65d",
  xboxMSAClientId: "ca634a9a-f102-4033-b081-3a4393a6f65d",
} as const;
```

the sisu browser URL is generated with
```ts
/**
 * Returns an MSA SISU login URL to redirect to.
 * @param search A string containing query params of the return url
 */
export function buildMsaLoginUrl(
  search: string,
  param: IBuildMsaLoginUrlParams = {}
) {
  const state = param.xboxState
    ? param.xboxState
    : environment["msaStateLoginParam"];
  const callbackUrl = param.returnURL
    ? param.returnURL
    : environment["url.xboxLoginReturnUrl"];
  const envName = getEnvironmentParam(search, callbackUrl);
  const ru = encodeURIComponent(callbackUrl + search + envName);
  const params = [
    `state=${state}`,
    `cobrandId=${environment.sisuCobrandId}`,
    environment["xboxSisuTestParam"],
    `tid=${environment.titleId}`,
    `ru=${ru}`,
    `aid=${environment.sisuAppId}`,
  ]
    .filter(Boolean)
    .join("&");
  return `${environment["endpoint.xboxAuthUrl"]}?${params}`;
}
```

So this should give some official insight on those endpoints. There's no need to use fixed IDs here, leave them for the user to put in.